scripts/benchmark.sh

```diff
@@ -44,14 +44,22 @@ benchmark_cmd() {
 run_xline() {
     cmd="/usr/local/bin/xline \
     --name node${1} \
-    --members ${MEMBERS}"
+    --members ${MEMBERS} \
+    --storage-engine memory \
```

Comment:
we should run benchmark with persistent storage

Reply:
Fixed